### DISCUSSION TITLE
Huawei P8 bootloader unlock error

### DISCUSSION THREAD

**Discussion by awalkh** [kitsuned/PotatoNV#136] [Q&A]
Thanks to @mashed-potatoes that has provided a free way to unlock Huawei smartphone bootloaders

I face error to unlock Huawei P8 bootloader 
My phone cpu is Hisilicon 936 it say time out during unlock process
![Screenshot_23](https://user-images.githubusercontent.com/115474683/194871964-bcc2228a-9ed3-4cfc-8053-0e55acf01574.png)
Any help is appreciated

**Comment by awalkh** [comment]
Now again show me this option
![Screenshot_24](https://user-images.githubusercontent.com/115474683/194994683-433cbecf-7b82-4d9b-8042-1be5c0f668cb.png)

**Reply by mashed-potatoes** [reply]
Hello.

Most likely, this topic is no longer relevant. Sorry.
If you still want to unlock this phone — [follow this](https://github.com/mashed-potatoes/PotatoNV/discussions/139#discussioncomment-4103495).
